### PR TITLE
add:create `cleanify` into `utils` modules

### PR DESCRIPTION
**Pull request by r1cardohj**
hey grey, 根据[代码厨房开源松](https://codekitchen.community/t/topic/1132)的指示，我已经创建了人生中第一个pr哈哈哈，我认为把这个函数叫做`cleanify`感觉还不错，有点类似flask里面的`jsonify`，我也加入了一些单元测试，麻烦有空的时候review一下啦，如果有不足的地方请提出来，我可以修改并从中受益，感谢。

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -6,6 +6,13 @@ Changelog
 
 Release date: -
 
+0.5.2
+-----
+
+Release date: 2023/12/09
```
Comment (greyli) [inline CHANGES.rst:12]: No need to add a specific date here, I will update it when I make the new release.

```suggestion
Release date: N/A
```

File: CHANGES.rst [hunk 2]
```diff
@@ -6,6 +6,13 @@ Changelog
...
+
+- add ``cleanify`` function to ``flask_ckeditor.utils``
```
Comment (greyli) [inline CHANGES.rst:14]:
```suggestion
- Add ``cleanify`` function to ``flask_ckeditor.utils`` for HTML sanity.
```

File: flask_ckeditor/utils.py [hunk 3]
```diff
@@ -1,8 +1,13 @@
 import os
 import uuid
-
+import warnings
 from flask import url_for
 
+try:
+    import bleach
+except ImportError:
+    warnings.warn('bleach is not installed,`cleanify` function will not be available')
```
Comment (greyli) [inline flask_ckeditor/utils.py:9]:
```suggestion
    warnings.warn('The "bleach" library is not installed, `cleanify` function will not be available.')
```

File: flask_ckeditor/utils.py [hunk 4]
```diff
@@ -15,3 +20,20 @@ def random_filename(old_filename):
     ext = os.path.splitext(old_filename)[1]
     new_filename = uuid.uuid4().hex + ext
     return new_filename
+
+
+def cleanify(text, *, allow_tags=None):
+    """clean the input from client, this function rely on bleach,
+
+
+    Args:
```
Comment (greyli) [inline flask_ckeditor/utils.py:29]: Use the docstring style that matches the [existing code](https://github.com/helloflask/flask-ckeditor/blob/master/flask_ckeditor/__init__.py#L20C7-L27).

File: flask_ckeditor/utils.py [hunk 5]
```diff
@@ -15,3 +20,20 @@ def random_filename(old_filename):
...
+        text (str): input
+        allow_tags (Iterable[str], optional): if you don't want to use default `allow_tags`
+            you can provide a Iterable which include html tag string like ['a', 'li',...]
+    """
+    if allow_tags:
+        return bleach.linkify(bleach.clean(text, tags=allow_tags))
+    default_allowed_tags = {'a', 'abbr', 'b', 'blockquote', 'code',
+                            'em', 'i', 'li', 'ol', 'pre', 'strong', 'ul',
+                            'h1', 'h2', 'h3', 'h4', 'h5', 'p'}
+    return bleach.linkify(bleach.clean(text, tags=default_allowed_tags))
```
Comment (greyli) [inline flask_ckeditor/utils.py:39]: You can simplify the code to this:

```suggestion
    default_allowed_tags = {'a', 'abbr', 'b', 'blockquote', 'code',
                            'em', 'i', 'li', 'ol', 'pre', 'strong', 'ul',
                            'h1', 'h2', 'h3', 'h4', 'h5', 'p'}
    return bleach.linkify(bleach.clean(text, tags=allow_tags or default_allowed_tags))
```

**Review by greyli — CHANGES_REQUESTED** [review]
Congratulations on your first PR!

The overall changes are good, but there are a few small things that could be improved, see the comments for details.

File: flask_ckeditor/utils.py (hunk 5 above)
Comment (r1cardohj) [reply flask_ckeditor/utils.py]: cool， i got it ！

Comment (r1cardohj) [reply flask_ckeditor/utils.py]: **Thank you for your patient review !** 
i has fixed those questions .

**Comment by r1cardohj** [comment]
**Thank you for your patient review !**
i has fixed those questions .

**Comment by greyli** [comment]
Merged, thanks! I made some improvements at https://github.com/helloflask/flask-ckeditor/pull/75/commits/16779d887cb3cdb2a327e0cda77a42deb0636fc5

**Pull request merged**